github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -401,7 +401,7 @@ presubmits:
       preset-docker-mirror-proxy: "true"
       preset-shared-images: "true"
     max_concurrency: 11
-    name: pull-kubevirt-e2e-k8s-1.20-cgroupsv2-rook-ceph
+    name: pull-kubevirt-e2e-k8s-1.20-sig-storage-cgroupsv2
```

Comment:
Is there a reason why we are using 1.20 here?

Reply:
No specific reason. It was initially configured for 1.20. Do you think we may need to switch to some other version?

Reply:
As 1.20 will reach EOL soon, I think we should bump to latest. Are there any objections, @iholder-redhat @vasiliy-ul ?

Reply:
Nope, seems fine by me :+1:

Reply:
No objections from my side. Switched to `1.23`

Reply:
Thanks! Taking this in now.